src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -427,6 +435,9 @@ public void removeDataBundle(DataBundle dataBundle) {
             }
         }
         accountsDb.deleteAccounts(dataBundle.accounts.values());
+
+        // Delete all admin emails
```

Comment:
Isn't our standard for comments to start with an upper case?

[Not explicitly stated here](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html) but all the examples suggest so.